racetime/templates/racetime/pops/entrant_row.html

```diff
@@ -31,6 +31,31 @@
                 {% endif %}
             </span>
         {% endif %}
+        {% if not race.is_done and entrant.user.youtube_channel %}
```

Comment:
I think the logic here should be changed so that we only show a Twitch icon or a YT icon. Probably give Twitch precedence, then fallback to YT if Twitch is offline but YT is live.

If the race is anonymous, showing whether a user has a Twitch or YT channel could compromise identities. For this I think it's best if we switch to using a generic icon, maybe [`hangout_video`](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:hangout_video:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=video&icon.size=24&icon.color=%231f1f1f)/[`hangout_video_off`](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:hangout_video_off:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=video&icon.size=24&icon.color=%231f1f1f)

Reply:
I went this way in case of dual streaming, I just copied the logic we're doing for twitch in these cases are we sure we want to hide information? That seems like a drift from what we're doing on twitch.

Reply:
Noted in discord, this is the only thing right now I don't have a solution for.